airbyte-webapp/license-check.js

```diff
@@ -0,0 +1,108 @@
+const checker = require('license-checker');
+const { promisify } = require('util');
+const { version } = require('./package.json');
+
+/**
+ * A list of all the allowed licenses that production dependencies can have.
+ */
+const ALLOWED_LICENSES = [
+  '(Apache-2.0 OR MPL-1.1)',
```

Comment:
@cgardens This list should be safe to be used with our license. Maybe you (or someone) want to double check this?

If you want I can also move this into a higher level place in the repo in case we want to use the same allow-list for other technologies too.